### PR TITLE
DOC-6428: max_parallelism description needs to be updated

### PR DESCRIPTION
**Pull request by simon-dew**
The following draft documentation is ready for review:

* [Index Partitioning › Performance Considerations › Max Parallelism](https://github.com/couchbase/docs-server/pull/1162/files)

Docs issue: [DOC-6428](https://issues.couchbase.com/browse/DOC-6428)

### CODE REVIEW

File: modules/n1ql/pages/n1ql-language-reference/index-partitioning.adoc [hunk 1]
```diff
@@ -639,9 +639,9 @@ The lost partitions cannot be repaired when the number of remaining nodes is les
 === Max Parallelism
 
 Along with aggregate pushdown optimization, an application can further enhance the aggregate query performance by computing aggregation in parallel for each partition in the index service.
-This can be achieved by specifying the parameter `max_parallelism` when issuing a query.
-The value for `max_parallelism` should match the number of partitions of the index.
-Note that when this is enabled, the index service uses more CPU and memory since the query traffic is increased according to the value set in the parameter `max_parallelism`.
+This can be controlled by specifying the parameter `max_parallelism` when issuing a query.
+In Couchbase Server 6.5 and later, `max_parallelism` is set by default to match the number of partitions of the index.
```
Comment (amarantha-k) [inline modules/n1ql/pages/n1ql-language-reference/index-partitioning.adoc]: For consistency, suggest rewording to "Starting with Couchbase Server 6.5, ..."

Comment (simon-dew) [reply modules/n1ql/pages/n1ql-language-reference/index-partitioning.adoc]: Done

**Review by deepkaran — APPROVED** [review]

**Review by amarantha-k — APPROVED** [review]
Lgtm
